pootle/apps/pootle_store/forms.py

```diff
@@ -373,3 +407,129 @@ def save(self, **kwargs):
             super(UnitCommentForm, self).save(**kwargs)
 
     return UnitCommentForm
+
+
+class UnitSearchForm(forms.Form):
+
+    initial = forms.BooleanField(required=False)
+    count = forms.IntegerField(required=False)
+    path = forms.CharField(
+        max_length=2048,
+        required=True)
+    uids = MultipleArgsField(
+        field=forms.IntegerField(),
+        required=False)
+    filter = forms.ChoiceField(
+        required=False,
+        choices=UNIT_SEARCH_FILTER_CHOICES)
+    checks = forms.MultipleChoiceField(
+        required=False,
+        widget=CommaSeparatedCheckboxSelectMultiple,
+        choices=check_names.items())
+    category = CategoryChoiceField(
+        required=False,
+        choices=CATEGORY_CODES.items())
+    month = forms.DateField(
+        required=False,
+        input_formats=['%Y-%m'])
+    sort = forms.ChoiceField(
+        required=False,
+        choices=UNIT_SEARCH_SORT_CHOICES)
+
+    user = forms.ModelChoiceField(
+        queryset=get_user_model().objects.all(),
+        required=False,
+        to_field_name="username")
+
+    search = forms.CharField(
+        required=False,
+        widget=forms.TextInput(attrs={
+            'size': '15',
+            'placeholder': _('Search'),
+            'title': _("Search (Ctrl+Shift+S)<br/>Type and press Enter to "
```

Comment:
k - remmed - i would like to get rid of pootle_misc.forms.SearchForm - but ill add this when that is done